docs/changelog.md

```diff
@@ -0,0 +1,4 @@
+```{eval-rst}
```

Comment:
Why are you using RST here? Just use a MyST-native include-directive. Maybe this is the reason the extension didn't work for you...

https://github.com/executablebooks/MyST-Parser/pull/469/files#diff-f58f9fe275eb1074a0b4a24a31f69d6fd72afb6a699a512b0bae9e0e1a97f5d5R74-R75

Reply:
nice you should totally stop talking only about rst in the docs – i read it as a given

Reply:
I guess with the current design of Towncrier, it's impossible to fix the link of Unreleased, huh?

Reply:
> nice you should totally stop talking only about rst in the docs – i read it as a given

Yeah, somebody recommended me to stop using rst in the function name years ago and I got to doing that just recently: https://github.com/sphinx-contrib/sphinxcontrib-towncrier/commit/ed76868cfff43091297a9c726815fadb85f3ef2f.

Reply:
One way could be allowing to specify a different template, plus it should be possible to specify a different config file with the recent versions already. It's a bit ugly to maintain, but might work. Another way could be allowing an arbitrary hook for post-processing whatever `towncrier --draft` spits out.

Reply:
I've moved the h1 into the template – think I've [nailed it](https://attrs--1067.org.readthedocs.build/en/1067/changelog.html) now. 🎉